packages/cli-kit/src/private/node/session/scopes.ts

```diff
@@ -35,7 +35,9 @@ function defaultApiScopes(api: API, systemEnvironment = process.env): string[] {
     case 'partners':
       return ['cli']
     case 'business-platform':
-      return ['destinations']
+      return isTruthy(systemEnvironment.USE_APP_MANAGEMENT_API)
+        ? ['destinations', 'store-management']
+        : ['destinations']
```

Comment:
I don't think we need this, as long as the scope is added to identity, this can be enabled for both cases.